backend/remote/backend_state.go

```diff
@@ -97,6 +99,11 @@ func (r *remoteClient) Delete() error {
 	return nil
 }
 
+// EnableForcePush to allow the remote client to overwrite state.
```

Comment:
In the TF codebase, we generally note in a comment when a function is implementing a particular interface; it makes things easier to sort out later on. For example: https://github.com/hashicorp/terraform/blob/master/state/remote/state.go#L62

Reply:
Oh that's nice. Thanks!